lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -34,6 +34,21 @@ class HttpApiEvents {
     this.provider = this.serverless.getProvider('aws');
     serverless.httpApiEventsPlugin = this;
 
+    serverless.configSchemaHandler.defineFunctionEvent('httpApi', {
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            method: { type: 'string' },
+            path: { type: 'string' },
+          },
+          required: ['path'],
+          additionalProperties: false,
```

Comment:
In scope of this PR let's not define any properties, and remove `additionalProperties: false`, to not break things.

Otherwise we run into significant task. I think it'll be good, after we have that one in, to cover each event with individual PR.

Reply:
I totally agree. I added it just for the sake of adding a use case for a friendly error message when unsupported event property is provided. Such error can't be shown for the very relaxed schema.

Here specifically I'll replace it with a relaxed schema.

As for that friendly message feature, I think I'll create a test with a schema where properties are defined to be more strict. It came to my mind only now. Sorry about that